src/AppCommon/Commands/ServiceControlCommand.cs

```diff
@@ -25,20 +25,28 @@ public static Command CreateCommand()
             IsRequired = true
         };
 
+        var tryUnsupportedVersionArg = new Option<bool>(TryUnsupportedVersionArgName)
+        {
+            Description = "This will allow you to try run against an unsupported version of service control.",
+            Arity = ArgumentArity.ZeroOrOne
+        };
+
```

Comment:
Add `IsHidden = true` option like https://github.com/Particular/Particular.EndpointThroughputCounter/blob/sc-tryUnsupportedVersion/src/AppCommon/Infra/SharedOptions.cs#L30-L31 so that it won't show up in command-line help, and then we only document it internally.